packages/animation-curve-block/src/constants.ts

```diff
@@ -0,0 +1,158 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { MenuBlock } from '@frontify/fondue';
+import { AnimationCurveType } from './types';
+
+export const DROPDOWN_MENU_ITEMS: MenuBlock[] = [
+    {
+        id: 'Standard',
```

Comment:
shouldn't the id's be camelcase?

Reply:
you 💯  right, changed all ids to camelCase